pom.xml

```diff
@@ -189,6 +189,43 @@
             <artifactId>configme</artifactId>
             <version>0.2.1</version>
         </dependency>
+
+        <!-- Database Connection Pool -->
+        <dependency>
+            <groupId>com.zaxxer</groupId>
+            <artifactId>HikariCP</artifactId>
+            <version>2.5.1</version>
+            <scope>compile</scope>
+            <exclusions>
+                <exclusion>
+                    <artifactId>slf4j-api</artifactId>
+                    <groupId>org.slf4j</groupId>
+                </exclusion>
+            </exclusions>
+            <optional>true</optional>
```

Comment:
I don't think so, but that's also how it is in AuthMe. Knowing me, it's probably something else I'm ignorant of. lol